tests/datastore/settings.q

```diff
@@ -0,0 +1,8 @@
+// IPC connection parameters
+.servers.CONNECTIONS:`segmentedtickerplant`rdb;
+
+// rmdtfromgetpar function in processes/rdb.q
+.rdb.rmdtfromgetpar:{[date] 
+    .rdb.rdbpartition:: .rdb.rdbpartition except date;
+    .lg.o[`rdbpartition;"rdbpartition contains - ","," sv string .rdb.rdbpartition];
+    }
```

Comment:
we shouldn't replicate code for the tests - we should be testing the _actual_ code that gets used. Reason being, if the code in `processes/rdb.q` gets changed down the line, there is very little chance anyone is going to remember that this test also needs to be updated, it will still be using the old version and could be "passing" even though the change broke something.

that may mean refactoring a little to move the function from `code/processes/rdb.q` to a file in `code/rdb/` so that it gets loaded in the test process.